en/brands.ftl

```diff
@@ -11,7 +11,6 @@
 -brand-name-apple = Apple
 -brand-name-creative-commons = Creative Commons
 -brand-name-facebook = Facebook
--brand-name-github = GitHub
```

Comment:
So this is an interesting wrinkle. These brand terms we're adding in https://github.com/mozilla-l10n/www-l10n/pull/41, but have not yet been merged to bedrock in https://github.com/mozilla/bedrock/pull/8958

Reply:
Yeah. Modifying these files directly in this repo and not in bedrock will result in this. Bedrock is meant to be the source of truth for the en files I believe.

Reply:
In future I would update this file in a separate PR in bedrock and let the automation open the PR like #41 .

Reply:
Ok, let's close this PR and add directly to bedrock